ansible/roles/svc_prometheus/tasks/install_prometheus.yml

```diff
@@ -0,0 +1,42 @@
+---
+# code: language=ansible
+# [Doc] List of all modules used
+# Apt https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html
+# File https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html
+# Get_Url https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html
+# Command https://docs.ansible.com/ansible/latest/collections/ansible/builtin/command_module.html
+# Shell https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html
+# Template https://docs.ansible.com/ansible/latest/collections/ansible/builtin/template_module.html
+
+- name: Install prometheus
+  ansible.builtin.apt:
+    name: prometheus
+    state: latest # noqa: package-latest
+    update_cache: true
+  become: true
+
+- name: Create prometheus user
+  ansible.builtin.user:
+    name: prometheus
+    state: present
+    shell: /bin/false
+    createhome: false
+  become: true
+
+- name: Create prometheus group
+  ansible.builtin.group:
+    name: prometheus
+    state: present
+  become: true
```

Comment:
If I'm not mistaken, a user and a group can be created in one task using the group directive. 
⚠️ group**s** != group

Reply:
It's possible to create user and group in one tasks

Reply:
I would opt for one task as it makes it simpler, more readable, and atomic. Also, creating the user before the group could leave the setup in an inconsistent state should one of these fail.

```
- name: Create prometheus user and group
  ansible.builtin.user:
    name: prometheus
    group: prometheus
    #groups: mygroup1,mygroup2 # Group membership
    state: present
    shell: /bin/false
    createhome: false
  become: true
```

Reply:
`ansible.builtin.user` can't create group, right ?